EmailNotifications/Email.cs

```diff
@@ -93,7 +93,7 @@ public void sendEmail()
                     EnableSsl = true
                 };
                 MailAddress from = new MailAddress("nmotsumi@retrorabbit.co.za", "Ntokozo Motsumi");
```

Comment:
Could you please put the from in the config as well.

Reply:
Information Put into the config file.